src/MoBi.Core/Commands/AddTopContainerCommand.cs

```diff
@@ -29,54 +24,14 @@ protected override ICommand<IMoBiContext> GetInverseCommand(IMoBiContext context
 
    public class RemoveTopContainerCommand : RemoveObjectBaseCommand<IContainer, MoBiSpatialStructure>
    {
-      private Cache<string, IObjectBase> _removedIds;
-
-      public RemoveTopContainerCommand(MoBiSpatialStructure parent, IContainer itemToRemove) : base(parent, itemToRemove, parent)
-      {
-      }
-
-      protected override void RemoveFrom(IContainer childToRemove, MoBiSpatialStructure spatialStructure, IMoBiContext context)
```

Comment:
We don't support any of this lol Removing a top container should not remove any neighbourhoods now. as they have soft links